components/applications-service/pkg/server/server.go

```diff
@@ -260,6 +260,26 @@ func (app *ApplicationsServer) GetServicesStats(c context.Context,
 	}, nil
 }
 
+// GetDisconnectedServices returns a list of disconnected services
+func (app *ApplicationsServer) GetDisconnectedServices(c context.Context,
+	request *applications.DisconnectedServicesReq) (*applications.ServicesRes, error) {
+
+	thresholdMinutes := request.GetThresholdMinutes()
+	if thresholdMinutes <= 0 {
+		return new(applications.ServicesRes),
+			status.Error(codes.InvalidArgument, "threshold must be greater than zero")
+	}
+	services, err := app.storageClient.GetDisconnectedServices(thresholdMinutes)
```

Comment:
Should this use the context?

Reply:
I’ll bring this up to the team, you are right and we should use a context.